src/frontend/Ast_to_Mir.ml

```diff
@@ -810,11 +807,11 @@ let%expect_test "gen quant" =
   print_s [%sexp (m.generate_quantities : stmt_loc list)] ;
   [%expect
     {|
-    ((IfElse (Var emit_generated_quantities__)
+    ((Decl (decl_adtype DataOnly) (decl_id mat)
+      (decl_type (SArray (SMatrix (Lit Int 10) (Lit Int 20)) (Lit Int 5))))
+     (IfElse (Var emit_generated_quantities__)
```

Comment:
Could you maybe remind me what this emit_generated_quantities__ flag is for again? Thanks!

Reply:
Like, in the code somewhere? Or just in the PR? If here, it's just a flag we allow to be passed to write_array (and maybe another method...):
```cpp
void stan_model::write_array(RNG& base_rng__,
                             std::vector<double>& params_r__,
                             std::vector<int>& params_i__,
                             std::vector<double>& vars__,
                             bool include_tparams__,
                             bool include_gqs__,
                             std::ostream* pstream__) const {
```